packages/devtools/lib/src/memory/memory.dart

```diff
@@ -680,11 +681,12 @@ class MemoryScreen extends Screen with SetStateMixin {
       ));
 
       table.setRows(instanceRows);
-    } catch (e) {
+    } catch (e, st) {
       framework.toast(
         'Problem fetching instances of ${row.classRef.name}: $e',
         title: 'Error',
       );
+      print(st);
```

Comment:
If we're going to output the stacktrace should probably output the toast message to console too?  Otherwise, hard to match.

Reply:
👍 will do